dango/testing/tests/indexing/candles.rs

```diff
@@ -497,14 +528,16 @@ async fn index_candles_with_both_market_and_limit_orders_one_minute_interval() -
         .fetch_all(clickhouse_context.clickhouse_client())
         .await?;
 
-    assert_that!(candle_1m.candles).has_length(1);
+    assert_that!(candle_1m.candles).has_length(2);
```

Comment:
@penso I'm uncertain about this change. I needed to break up the order creation into more txs due to the new market order logic. Is it reasonable that two candles would be indexed then? What is the block time here? I assume 1m means 1 minute candles, is that correct?

Reply:
`candle_1m` is just the name, but if you look at the query builder it does mention `CandleInterval::OneMinute` so yes.

You shouldn't have 2 candles unless the blocks are created over longer than a minute apart, or unless you have more than a single pair.